docs/modules/SDDs/pages/0006-commodore-configuration-catalog-compilation.adoc

```diff
@@ -82,33 +82,33 @@ Component versions can be any git https://git-scm.com/docs/gitglossary#Documenta
 
 * Components are versioned as part of the inventory repositories, which comes from VSHN defaults and the cluster / customer specific repository
 * Each component is managed in an individual Git repository, comparable to a Puppet module.
-* Components bundle the Kapitan component templates (for example Jsonnet) with the component class that will be referenced in the Kapitan inventory and optional postprocessing filters which are applied on the compiled Kapitan catalog. The engine downloads the component repository into dependencies  and symlinks the component class into inventory/classes/components  to make the class available in the Kapitan inventory structure.
```

Comment:
Please be consistent whether or not to include a tailing `/` for notating directories.